uberenv.py

```diff
@@ -1202,6 +1199,9 @@ def main():
 
         return 0
 
+    # Show the spec for what will be built
+    env.show_info()
+
```

Comment:
Looking at this, I wonder:
If the result of the concretization depends on what is already installed (--reuse), shouldn’t the concretization be done after the mirrors and upstream are set ?
In other words, I worry that reordering things that way could show a different concretization than what will actually be built.
What do you think?

Reply:
This is a change back to what is already in uberenv. This call to `show_info()` is not modified by this PR. I agree however that `show_info()` might make more sense to be called after upstreams / mirrors are set.